pkg/gadgets/biotop/gadget.go

```diff
@@ -0,0 +1,202 @@
+// Copyright 2019-2021 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package biotop
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/kinvolk/inspektor-gadget/pkg/gadgets"
+	biotoptracer "github.com/kinvolk/inspektor-gadget/pkg/gadgets/biotop/tracer"
+	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/biotop/types"
+
+	gadgetv1alpha1 "github.com/kinvolk/inspektor-gadget/pkg/apis/gadget/v1alpha1"
+)
+
+const (
+	maxRowsDefault  = 20
+	intervalDefault = 1
+	sortByDefault   = types.ALL
+)
+
+type Trace struct {
+	resolver gadgets.Resolver
+
+	started bool
+	tracer  *biotoptracer.Tracer
+}
+
+type TraceFactory struct {
+	gadgets.BaseFactory
+}
+
+func NewFactory() gadgets.TraceFactory {
+	return &TraceFactory{
+		BaseFactory: gadgets.BaseFactory{DeleteTrace: deleteTrace},
+	}
+}
+
+func (f *TraceFactory) Description() string {
+	t := `biotop shows command generating block I/O, with container details.
+
+The following parameters are supported:
+ - interval: Output interval, in seconds. (default %d)
+ - max_rows: Maximum rows to print. (default %d)
+ - sort: The field to sort the results by (%s). (default %s)`
```

Comment:
Isn't it `sort_by`? I noticed the same problem on the filetop gadget, could you please fix both.